aptos-move/e2e-move-tests/src/tests/transaction_context.data/sources/transaction_context_test.move

```diff
@@ -0,0 +1,28 @@
+module 0x1::transaction_context_test {
+    use std::vector;
+    use aptos_framework::transaction_context::generate_unique_address;
+    use std::features;
+
+    /// When checking the value of aggregator fails.
+    const ENOT_UNIQUE: u64 = 20;
+
+    public entry fun create_many_auids(_account: &signer, count: u64) {
+        if (features::auids_enabled()) {
+            let auids: vector<address> = vector<address>[];
+            let i: u64 = 0;
+            while (i < count) {
+                i = i+1;
+                vector::push_back(&mut auids, generate_unique_address());
+            };
+            i = 0;
+            while (i < count - 1) {
```

Comment:
This validation looks like something that should be done inside a unit test for the transaction_context.move module, not here?

Reply:
Moved the the test from e2e-move-tests to a unit test in transaction_context.move.